src/styles/blocks/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, Arial, sans-serif;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__icon {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  margin-bottom: 4px;
+  color: var(--color-dark);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  color: var(--color-gray);
+  font-weight: 400;
```

Comment:
400 is a default value, if not set it to default, no need to set it everywhere and fix it everywhere